model/host/db.go

```diff
@@ -205,18 +205,35 @@ func IdleEphemeralGroupedByDistroID() ([]IdleHostsByDistroID, error) {
 func runningHostsQuery(distroID string) bson.M {
 	query := IsLive()
 	if distroID != "" {
-		key := bsonutil.GetDottedKeyName(DistroKey, distro.IdKey)
-		query[key] = distroID
+		query[bsonutil.GetDottedKeyName(DistroKey, distro.IdKey)] = distroID
 	}
 
 	return query
 }
 
+func provisioningHostsQuery(distroID string) bson.M {
+	query := bson.M{StatusKey: bson.M{"$in": []string{evergreen.HostBuilding, evergreen.HostStarting}}}
```

Comment:
This should be a variable in https://github.com/evergreen-ci/evergreen/blob/master/globals.go#L410-L462.